modules/git/repo_language_stats.go

```diff
@@ -46,3 +48,17 @@ func mergeLanguageStats(stats map[string]int64) map[string]int64 {
 	}
 	return res
 }
+
+func TryReadLanguageAttribute(attrs map[string]string) optional.Option[string] {
+	language := AttributeToString(attrs, AttributeLinguistLanguage)
+	if language.Value() == "" {
+		language = AttributeToString(attrs, AttributeGitlabLanguage)
+		if language.Has() {
+			raw := language.Value()
+			if idx := strings.IndexByte(raw, '?'); idx >= 0 {
```

Comment:
I see it is from old code ... I am quite curious that why checking `?` here ...  feel free to add a comment or dismiss this question if it is still unclear.

Reply:
I've seen in the tests that the format can be something like
`*.md gitlab-language=Go?parent=Markdown`.
But yes, it is a weird mechanism…

Reply:
Perhaps we should add a comment for it.

Reply:
The syntax is shown here: https://docs.gitlab.com/ee/user/project/highlighting.html#override-syntax-highlighting-for-a-file-type
> You can also extend the highlighting with Common Gateway Interface (CGI) options, such as:

Whatever that means.